CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Bug Fixes
 
+- Enabled Flake8-bugbear linter, fixing potential bugs, improving code quality and detecting design issues in advance ([#3854](https://github.com/pybamm-team/PyBaMM/issues/3854))
```

Comment:
We can skip this note in the CHANGELOG. It is not a user-facing bug fix and affects just developers